modules/conflict_resolver/jsx/fix_conflict_form.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * This file contains the React component for the fix conflict form
+ * to be rendered in the cells of the conflict_resolver datatable.
+ */
+import {Component} from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * The fix FixConflictForm renders a <form> within a <td>. The form as a select
+ * dropdown with the two possible answers as values; plus an empty value (default).
+ * On change events triggers a POST request to /conflict_resolver/unresolved with
+ * the conflict_id and the selected value as payload.
+ *
+ * It is possible to change the value multiple time, each time sending a new POST
+ * request to the same endpoint. After the first request, the empty option of the
+ * dropdown is removed to prevent the user from sending a POST request with an empty
+ * value.
+ *
+ * Knowned issue: When sorting the datatable, previouly fixed conflicts are
+ * considered unresolved; there is no green checkmark beside the dropdown anymore.
+ */
+class FixConflictForm extends Component {
+  /**
+   * Constructor
+   *
+   * @param {object} props - The provided properties.
+   */
+  constructor(props) {
+    super(props);
+
+    this.fix = this.fix.bind(this);
+  }
+
+  /**
+   * Callback for the select dropdown onChange event.
+   *
+   * Sends a POST request to /conflict_resolver/unresolved containing the
+   * conflict_id and the selected value name.
+   *
+   * If the request is successful, a green checkmark is displayed in a <span>
+   * beside the dropdown. On error, a red cross will be displayed as well as a
+   * sweetalert (swal) with the error message.
+   *
+   * @param {Event} e - The onChange event.
+   */
+  fix(e) {
```

Comment:
I would suggest having this function alter the DOM via a state change, rather than altering the element directly. That way you can also use the LORIS SelectElement rather than a custom element.

Reply:
Would you care to propose something?

Reply:
You suggestion must:
1. send a port request to `/conflict_resolver/unresolved` with `{conflictid: conflictid, correctanswer: correctanswer}`
2. display feedback to the user for each resolved conflict.
3. remove the empty option in the select options when a conflict is resolved.

Reply:
Sure.

1. This wouldn't have to change. The `fix` function could still post the request the same way you are currently doing it.
2. The `fix` function would call a function that will have been passed by the parent via props and pass an `id` that indicates which row of the datatable it corresponds to. This callback function would then trigger a state change in the parent which would display the feedback to the user for that given datatable row.
3. You could use the same state form (2) to conditionally remove the empty option when a conflict has been resolved. I believe there's a prop you can pass to select elements that determine whether there is an empty option or not.

Reply:
@HenriRabalais Would you care to propose something? (I mean code)

Reply:
You want me to commit code to your PR?

Reply:
Sending a PR to my branch would do the trick.